### ISSUE TITLE
Buttons not vertically aligned

### ISSUE THREAD

**Issue by naasking** [franciscop/picnic#138]
Buttons in a grid don't seem to vertically align properly:

http://codepen.io/anon/pen/OpyLdQ

**Comment by franciscop** [comment]
Yes, I've had button's margin come back to byte me in several places, will remove them in the next major version, thanks for the report
